src/main/java/com/capitalone/dashboard/service/AutoDiscoveryServiceImpl.java

```diff
@@ -44,6 +44,7 @@ public AutoDiscoveryServiceImpl(AutoDiscoveryRepository autoDiscoveryRepository,
     @Override
     public AutoDiscovery save(AutoDiscoveryRemoteRequest request) throws HygieiaException {
         String autoDiscoveryId = request.getAutoDiscoveryId();
+        request.getMetaData().getProperties();
```

Comment:
Is this info ever used?

Reply:
this line of code is not required. Please remove it

Reply:
It has been removed now